### ISSUE TITLE
Add alt text to user github profile image

### ISSUE THREAD

**Issue by ankushChatterjee** [jenkoian/hacktoberfest-checker#226]
According to Axe accessibility audit, images must have an alt text. But this image does not have one.

Should I go ahead for a PR?

**Comment by jenkoian** [comment]
Fixed in #227
